spec/chunk_spec.js

```diff
@@ -0,0 +1,39 @@
+import { chunk } from '../src/chunk'
+
+describe('chunk', ()=> {
+  it('returns array of elements in an array when chunk size is 1', () => {
+    const chunkThis = [1,2,3,4,5]
+    expect(chunk ( chunkThis, 1 ) ).toEqual( [[1],[2],[3],[4],[5]] )
+   })
+
+  it('returns array of elements in an array of specified chunk size', () => {
+    const chunkThis2 = [1,2,3,4,5]
+    expect(chunk ( chunkThis2, 2 ) ).toEqual( [[1,2] , [3,4] , [5]] )
+  })
+
+  it('returns array of elements in an array of specified chunk size, with remainder', () => {
+    const chunkThis3 = [1,2,3,4,5]
+    expect(chunk ( chunkThis3, 3 ) ).toEqual( [[1,2,3], [4,5]] )
+  })
+    
+  it('returns array of elements from an object of specified chunk size if ', () => {
```

Comment:
Was trying to parse this test description - what does the last if mean? (Is there something missing?)